client/coral-sign-in/containers/SignInContainer.js

```diff
@@ -119,8 +119,7 @@ class SignInContainer extends Component {
     }
   }
 
-  handleSignIn(e) {
-    e.preventDefault();
```

Comment:
It looks like e.preventDefault() messes with `onmousedown` in firefox in a way that's causing this problem. It's a good practice, but if it's breaking firefox then we should be without it.